app/controllers/mixins/ems_common_angular.rb

```diff
@@ -114,6 +114,9 @@ def create
     end
 
     def get_task_args(ems)
+      if ems.respond_to?(:validate_authentication_args)
+        return ems.validate_authentication_args(params)
+      end
```

Comment:
There are a couple of `when` cases below this conditional -- and they return more data than just `[params[:default_userid], MiqPassword.encrypt(params[:default_password])]`

Would the `ems` that passes  this `if ems.respond_to?(:validate_authentication_args)` conditional never be one of those `when` cases?